magstock/static/food_menu.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
This is an example of us adding a new static file which doesn't even exist in core Uber.